tests/test_main.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import json
+from main import app
+from fastapi.testclient import TestClient
+
+client = TestClient(app)
+
+
+def test_there_is_data_in_json_file():
```

Comment:
We do not test that a file contains data, this is not a unit test. It tests not functionality.